src/tiled/mainwindow.ui

```diff
@@ -591,6 +592,17 @@
     <string>Edit Commands...</string>
    </property>
   </action>
+  <action name="actionClearView">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Clear View</string>
+   </property>
+   <property name="shortcut">
+    <string>Q</string>
```

Comment:
I don't think the Tab for navigation is actually working very well for navigating the global UI. It's hard to see where the focus is and eventually you simply get stuck in the tileset view. I'm not surprised that other applications have chosen to use Tab for something more useful, and I think we can do the same with Tiled.